packages/terra-list/src/List.jsx

```diff
@@ -8,6 +8,25 @@ import styles from './List.module.scss';
 const cx = classNamesBind.bind(styles);
 
 const propTypes = {
+  /**
+   * References a visible or visually hidden text element
+   * on the page that conveys information about the list
+   * and how to interact with it.
+   */
+  ariaDescribedBy: PropTypes.string,
+  /**
+    * String that provides information about the list and how to interact
+    * with it. When used with `ariaDescribedBy` prop, `ariaDescribedbBy`
+    * takes precedence in defining the accessible description property.
+    */
+  ariaDescription: PropTypes.string,
+  /**
+   * References a visible or visually hidden text element
+   * on the page that conveys information about the list
+   * and how to interact with it. Can be used alongside
+   * ariaDescribedBy or ariaDescription.
+   */
+  ariaDetails: PropTypes.string,
```

Comment:
Since these descriptions get added to the props table on the documentation, I would prefer to add a bit more information for usage and clarity. Maybe, we can link the aria documentation?

e.g.
```suggestion
  /**
   * References a visible or visually hidden text element
   * on the page that conveys information about the list
   * and how to interact with it. Maps to [area-describedby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby)
   */
  ariaDescribedBy: PropTypes.string,
  /**
    * String that provides information about the list and how to interact
    * with it. When used with `ariaDescribedBy` prop, `ariaDescribedbBy`
    * takes precedence in defining the accessible description property. Maps to [area-description](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description)
    */
  ariaDescription: PropTypes.string,
  /**
   * References a visible or visually hidden text element
   * on the page that conveys information about the list
   * and how to interact with it. Can be used alongside
   * ariaDescribedBy or ariaDescription. Maps to [area-details](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-details)
   */
  ariaDetails: PropTypes.string,
```

Reply:
[250e017](https://github.com/cerner/terra-core/pull/3688/commits/250e0179b89df8feb46bc63796388b7619fdbef6)